### PR TITLE
Reorg local blockchain run scripts

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Reorg local blockchain run scripts
- Add `ENABLE_EVENT_HANDLER=true \` to local blockchain node 2 for ain-js event manager testing

### CODE REVIEW

File: start_local_blockchain.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+printf "\n[[[[[ start_local_blockchain.sh ]]]]]\n\n"
```
Comment (liayoo) [inline start_local_blockchain.sh:3]: Could you leave a note at the top about the event handler node (how it's different from start_servers_local.sh)?

**Review by liayoo — APPROVED** [review]
LGTM with a comment. Thanks!

File: start_local_blockchain.sh (hunk 1 above)
Comment (platfowner) [reply start_local_blockchain.sh:6]: Just done, thanks!

**Comment by platfowner** [comment]
Thanks for the fast review, now merging.. @liayoo

**Pull request merged**